apps/hub-app/src/components/Header.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Dropdown, ParMd } from '@daohaus/ui';
+import { BiCommand, BiChevronDown } from 'react-icons/bi';
```

Comment:
I do -- I'm not sure if we landed on a complete consensus about it. Good callout -- there aren't exact matches in the remix icons library, so that would be a compromise too. These ones seem closer to the Figma. We should bring this up